libs/core/src/components/calendar/calendar.ts

```diff
@@ -58,7 +58,7 @@ export type CustomizedDate = {
  */
 @gdsCustomElement('gds-calendar')
 export class GdsCalendar extends GdsElement {
-  static styles = [style]
+  static styles = [tokens, style]
```

Comment:
Was thinking about this the otherday: shouldn't we remove this implementation of tokens and require the use of `<gds-theme>`?

"Forcing" the use of `<gds-theme>` I think helps with implementation of dark mode!

Reply:
Opened a separate PR for this since is a significant change, let's take a decision tomorrow and then merge it into this! 

PR: #1514